addOns/reports/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 - Maintenance changes.
+- Changed to handle changes related to contexts having multiple URLs.
```

Comment:
`Handle multiple context URLs in automation`? It's not clear that's related to automation otherwise. (Same comment for AJAX Spider.)